backend/ltsa/Models/Altos1ChargeCorrection.cs

```diff
@@ -0,0 +1,116 @@
+/* 
```

Comment:
All models are generated by swagger, but many files needed to be hand edited to correct errors in the swagger spec. Also, the generated files contained a large number of unused utility methods that I've removed (to prevent a code coverage nightmare).